Sources/TuistSupport/Utils/FileHandler.swift

```diff
@@ -165,6 +175,13 @@ public class FileHandler: FileHandling {
             throw FileHandlerError.invalidTextEncoding(at)
         }
     }
+    
+    public func readPlistFile<T: Decodable>(_ at: AbsolutePath) throws -> T {
+        guard let data = fileManager.contents(atPath: at.pathString) else {
+            throw FileHandlerError.fileNotFound(at)
+        }
+        return try PropertyListDecoder().decode(T.self, from: data)
```

Comment:
Instead of creating instances of the decoder every time this method is called, what about having an instance variable in the `FileHandler` class:

```
let propertyListDecoder = PropertyListDecoder()
```

Reply:
as it has got single usage currently, I've resorted to a local instance it could be declared as an instance.